setup.py

```diff
@@ -35,8 +35,8 @@ def pkgconfig_installed_check(lib, required_version, default):
     liblz4_found = pkgconfig_installed_check('liblz4', LZ4_REQUIRED_VERSION, default=False)
 
 # Establish if we want to build experimental functionality or not.
-experimental = os.environ.get("PYLZ4_EXPERIMENTAL", False)
-if experimental is not False:
+experimental = os.environ.get("PYLZ4_EXPERIMENTAL", True)
```

Comment:
I had to change this to have the tests pass.

Reply:
Nope, we're not enabling this for release - the code is broken, the tests take forever to execute, and I do not have the bandwidth to support it. Until a maintainer steps up to maintain the streaming code it will remain unbuilt by default. This code was the result of a drive by contribution, and I am not very convinced of the code design.

If you needed to enable this to pass the tests, I suspect you're not running the tests via tox as expected. Simply running `pytest` to invoke all the tests will hit the problem you describe, and this is why using tox is the correct way.

Reply:
Hi @jonathanunderwood  - I had to enable this to have the tests pass correctly in CI.